src/components/NavigationMailbox.vue

```diff
@@ -174,7 +174,10 @@
 				{{ t('mail', 'Delete mailbox') }}
 			</ActionButton>
 		</template>
-		<AppNavigationCounter v-if="showUnreadCounter && mailbox.specialRole !== 'trash'" slot="counter">
+		<AppNavigationCounter v-if="showUnreadCounter && subCounter" slot="counter">
```

Comment:
wouldn't this only show the unread counter if there was an unread item in the submailbox?

Reply:
this is for unread messages in a child. the one for the parent is in the else-if